src/main/resources/templates/db_scripts/schema.json

```diff
@@ -3,21 +3,13 @@
     {
       "tableName": "template",
       "withMetadata": true,
-      "pkColumnName": "_id",
-      "generateId": false,
       "withAuditing": false,
       "uniqueIndex": [
```

Comment:
I don't think you need to keep a unique index on **id** field. It is primary key and it will be unique and indexed by default

Reply:
Exactly.